sdk/identity/Azure.Identity/samples/OtherCredentialSamples.md

```diff
@@ -1,15 +1,19 @@
 # AzurePipelinesCredential Example
 
 This example demonstrates authenticating the `SecretClient` using the `AzurePipelinesCredential` in an Azure Pipelines environment with [service connections](https://learn.microsoft.com/azure/devops/pipelines/library/service-endpoints).
+***Note:*** This credential requires environment variables which are only available in the context of an Azure [PowerShell](https://learn.microsoft.com/azure/devops/pipelines/tasks/reference/powershell-v2?view=azure-pipelines) or [Azure CLI](https://learn.microsoft.com/azure/devops/pipelines/tasks/reference/azure-cli-v2?view=azure-pipelines) task.
+
+More information about the System.AccessToken can be found [here](https://learn.microsoft.com/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#systemaccesstoken).
 
 ```C# Snippet:AzurePipelinesCredential_Example
-// Replace the following values with the actual values for the service connection.
-string clientId = "<service_connection_client_id>";
-string tenantId = "<service_connection_tenant_id>";
-string serviceConnectionId = "<service_connection_id>";
+// Replace the following values with the actual values for the System.AccessToken and the details for your service connection.
+string systemAccessToken = "<the value of System.AccessToken>";
```

Comment:
Should we have samples demonstrate retrieving the system access token from an environment variable instead (i.e. `Environment.GetEnvironmentVariable("SYSTEM_ACCESSTOKEN")`), then passing that in?